config/routes.rb

```diff
@@ -5,5 +5,9 @@
   post 'sessions/authenticate', to: 'sessions#authenticate', as: 'authenticate'
   resources :users, except: :destroy
   get 'users/edit_password/:id', to: 'users#edit_password', as: 'edit_password'
-  post 'users/update_password', to: 'users#update_password', as: 'update_password'  
+  post 'users/update_password', to: 'users#update_password', as: 'update_password'
+  get 'users/add_image/:id', to: 'users#add_image', as: 'add_image'
```

Comment:
What's the need for `:id` here?

Reply:
ma'am to specify the id of the user to which the image has to be attached

Reply:
Then it should be like
```suggestion
  get 'users/:id/add_image', to: 'users#add_image', as: 'add_image'
```